app/models/order_import.rb

```diff
@@ -131,8 +131,15 @@ def handle_save_nothing_on_error # TODO refactor and rename
   end
 
   def import_row(row) # TODO refactor
-    row_importer = OrderRowImporter.new(row, self)
-    row_importer.import
+    begin
+      row_importer = OrderRowImporter.new(row, self)
+      row_importer.import
+    rescue => e
+      ActiveSupport::Notifications.instrument('external_error',
```

Comment:
Let's use something other than `external_error` for the name of this. That name made sense for GS since we were using it for errors relating to using external services/APIs. Maybe `background_error` unless you have something better.